frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.html

```diff
@@ -86,6 +86,34 @@ <h2 class="heading3 push-prev">Write Policy</h2>
                 <wizard-controls></wizard-controls>
             </div>
         </section>
+
+        <section class="column greedy">
+            <div class="column greedy pad">
+                <h2 class="heading3">Caching Configuration</h2>
+                <p class="push-next">
+                    Caching configuration controls how data is cached in local storage
```

Comment:
So perhaps we need to have a ui-toggle that opens this section on explicitly in order to enable caching?

Reply:
I will make it optional. I am not UI developer. It would have to be polished.